test/JDBC/expected/BABEL-736.out

```diff
@@ -0,0 +1,245 @@
+SELECT degrees(10)
+GO
+~~START~~
+int
+572
+~~END~~
+
+
+SELECT degrees(NULL)
+GO
+~~START~~
+float
+<NULL>
+~~END~~
+
+
+SELECT degrees(9999*9999)
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Arithmetic overflow error converting expression to data type int)~~
+
+
+SELECT degrees(9999.0 * 9999.0)
```

Comment:
this result is different from sql server, and is not part of your implementation, please remove this case